src/NCrypt.Tests/NCrypt.cs

```diff
@@ -15,4 +15,26 @@ public void OpenStorageProvider()
         {
         }
     }
+
+    [Fact(Skip = "fails, and I don't know how to fix it")]
```

Comment:
Does it always fail or just on appveyor ? If only on appveyor can it be ignored only in this case ?